CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# 0.0.6 (August 27, 2018)
+
+* (#)
```

Comment:
What does it stand for?

Reply:
It's a WIP. I will add more PRs (if they will be merged) until August 27.